plugins/dummy/src/dummy.cpp

```diff
@@ -155,4 +152,33 @@ Source::destroy(void* p)
 	delete static_cast<VideoSource*>(p);
 }
 
+void
+Output::operator()(const std::string& metadata, const dto::ImageView*) noexcept
+{
+	std::cout << "Received result: " << metadata << '\n';
```

Comment:
I am considering a forced delay between result prints here, since there is one per frame.